src/frontend/components/Firefighter/index.js

```diff
@@ -10,6 +10,10 @@ const label = {
 };
 
 class Firefighter extends PureComponent {
+  preventDefault = evt => {
```

Comment:
A little bit redundant no?

Reply:
it's just because I need the function in order to use in the context menu *no arrow functions on render methods*

Reply:
Right